src/services/s3/getS3Object.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  S3Client,
+  GetObjectCommand,
+  GetObjectCommandInput
+} from '@aws-sdk/client-s3'
+import { Readable } from 'stream'
+import { getEnv } from '../../utils/helpers'
+
+export const getS3ObjectAsStream = async (
```

Comment:
Might want to name this file `getS3ObjectAsStream.ts`

Reply:
done!